src/Command/Issue/IssueTakeCommand.php

```diff
@@ -31,19 +32,18 @@ protected function configure()
             ->setDescription('Takes an issue')
             ->addArgument('issue_number', InputArgument::REQUIRED, 'Number of the issue')
             ->addArgument('base_branch', InputArgument::OPTIONAL, 'Name of the base branch to checkout from')
-            ->addOption(
-                'remote',
-                null,
-                InputOption::VALUE_REQUIRED,
-                'Name of the Git remote (default is origin)',
-                null
-            )
             ->setHelp(
                 <<<EOF
 The <info>%command.name%</info> command takes an issue from issue tracker repository list:
 
     <info>$ gush %command.name% 3</info>
 
+In practise this will add the organization as remote (if not registered already),
+then <comment>git checkout base_branch</comment> and create a new branch that is equal to the issue-number + title.
+
+After you are done you can open a new pull-request using the <info>$ gush pull-request:create</info> command.
+<comment>Note:</comment> you must push the branch before opening a pull-request!
```

Comment:
I actually got an idea for that as well ;) Oh how many times I forgot to push before opening the pull-request :scream: